src/runtime/audience-action-flow.js

```diff
@@ -293,6 +298,15 @@ export class AudienceActionFlow {
     // @TODO(justinchou): execute callback with setOnInterventionComplete
     // then check for success
     const gaLoggingSuccess = this.logSurveyDataToGoogleAnalytics(request);
+    if (!gaLoggingSuccess) {
+      this.deps_
+        .eventManager()
+        .logSwgEvent(
+          AnalyticsEvent.EVENT_SURVEY_DATA_TRANSFER_FAILED,
+          /* isFromUserAction */ true
```

Comment:
Is this event categorized as `isFromUserAction == true` ?

Reply:
I think it's `false` as failure per se is not an user action